app/controllers/application_controller.rb

```diff
@@ -1797,6 +1797,15 @@ def task_supported?(typ)
     end
 
     vms = VmOrTemplate.where(:id => vm_ids)
+    if "migrate" == typ
+      if vms.map(&:ext_management_system).uniq.compact.any? do |ems|
+        ems.respond_to?("supports_#{typ}_for_all?") && !ems.send("supports_#{typ}_for_all?", vms)
```

Comment:
please see my comment on the other PR https://github.com/ManageIQ/manageiq/pull/13077#discussion_r96680257

So, not sure about this.